packages/switchToggle/src/Default/Default.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { Form, ThemeProvider } from 'react-bootstrap';
+import {
+  SwitchToggleDefaultProps,
+  SwitchToggleContainerProps,
+} from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: SwitchToggleDefaultProps) => {
+  const baseClass = 'switchToggle';
+  const { switchtoggle } = props;
+  const prefix = props.prefix || '';
+
+  const [toggle, setToggle] = React.useState<boolean>(false);
+
+  const locals: SwitchToggleDefaultProps = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'theme',
+          },
+          {
+            fields: ['appearance'],
+            value: 'Appearance',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    [
+      'prefix',
+      'theme',
+      'appearance',
+      'Size',
+      'switchToggle',
+      'label',
+      'id',
+      'type',
+    ]
+  );
+
+  return (
+    <>
+      <ThemeProvider
+        prefixes={{
+          'form-check': 'owlui-form-check',
+          'form-switch': 'owlui-form-switch',
+          'form-check-input': 'owlui-form-check-input',
+          label: 'owlui-label',
+        }}
+      >
+        <Form {...locals}>
+          <label
+            htmlFor="owlui-custom-switch"
+            className={`label p-2 ${toggle ? 'active-switch' : ''}`}
+          >
+            <span className="m-2" aria-hidden="true">
+              Reduced Motion
+            </span>
+            <Form.Check
```

Comment:
You can remove most of these props. We only actually need the `type="switch"` all the others will be passed as props to `locals` when using the switch. It should look like:
```tsx
    <ThemeProvider
      prefixes={{
        'form-check': 'owlui-form-check',
        'form-switch': 'owlui-form-switch',
        'form-check-input': 'owlui-form-check-input',
        'form-check-label': 'owlui-form-check-label',
      }}
    >
      <Form.Check {...locals} type="switch" />
    </ThemeProvider>
``` 
This way you don't need to manage state within the component, which will reduce a couple more lines of code.